es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = 3.1415926535 * (this.radius * this.radius);
+    this.perimeter = 2 * 3.1415926535 * this.radius;
```

Comment:
You shouldn't be creating `area` & `perimeter` properties directly, you should be creating them from `super`